tests/bluetooth/bsim/nrf_auraconfig/tester/src/nrf_auraconfig_test.c

```diff
@@ -0,0 +1,1164 @@
+				expected.subgroups[i].program_info, dut.subgroups[i].program_info);
+			verdict = false;
+		} else {
+			LOG_DBG("\tProgram info: " COLOR_GREEN "Success" COLOR_RESET);
+		}
+
+		if (expected.subgroups[i].num_bis != dut.subgroups[i].num_bis) {
+			LOG_WRN("\tExpected num_bis: %d, received: %d",
+				expected.subgroups[i].num_bis, dut.subgroups[i].num_bis);
+			verdict = false;
+		} else {
+			LOG_DBG("\tNum_bis: " COLOR_GREEN "Success" COLOR_RESET);
+		}
+
+		for (size_t j = 0U; j < dut.subgroups[i].num_bis; j++) {
+			if (expected.subgroups[i].locations[j] != dut.subgroups[i].locations[j]) {
+				LOG_WRN("\tExpected location: %d, received: %d",
+					expected.subgroups[i].locations[j],
+					dut.subgroups[i].locations[j]);
+				verdict = false;
+			} else {
+				LOG_DBG("\tLocation: " COLOR_GREEN "Success" COLOR_RESET);
+			}
+		}
+	}
+
+	if (verdict == true) {
+		bst_result = Passed;
+		LOG_INF("Verdict: " COLOR_GREEN "SUCCESS" COLOR_RESET);
+	} else {
+		bst_result = Failed;
+		LOG_WRN("Verdict: " COLOR_RED "FAIL" COLOR_RESET);
+	}
+}
+
+void nac_test_loop(void)
+{
+	int err;
+	uint32_t sync_bitfield;
+
+	err = k_sem_take(&sem_broadcaster_found, K_FOREVER);
+	if (err != 0) {
+		LOG_DBG("sem_broadcaster_found timed out, resetting");
+	}
+
+	err = bt_le_scan_stop();
+	if (err != 0) {
+		LOG_DBG("bt_le_scan_stop failed with %d, resetting", err);
+	}
+
+	LOG_DBG("Attempting to PA sync to the broadcaster with id 0x%06X",
+		broadcaster_broadcast_id);
+
+	err = pa_sync_create();
+	if (err != 0) {
+		LOG_DBG("Could not create Broadcast PA sync: %d, resetting", err);
+	}
+
+	LOG_DBG("Waiting for PA synced");
+
+	err = k_sem_take(&sem_pa_synced, SEM_TIMEOUT);
+	if (err != 0) {
+		LOG_DBG("sem_pa_synced timed out, resetting");
+	}
+
+	LOG_DBG("Broadcast source PA synced, creating Broadcast Sink");
+
+	err = bt_bap_broadcast_sink_create(pa_sync, broadcaster_broadcast_id, &broadcast_sink);
+	if (err != 0) {
+		LOG_DBG("Failed to create broadcast sink: %d", err);
+	}
+
+	LOG_DBG("Broadcast Sink created, waiting for BASE");
+
+	err = k_sem_take(&sem_base_received, SEM_TIMEOUT);
+	if (err != 0) {
+		LOG_DBG("sem_base_received timed out, resetting");
+	}
+
+	LOG_DBG("BASE received, waiting for syncable");
+
+	err = k_sem_take(&sem_syncable, SEM_TIMEOUT);
+	if (err != 0) {
+		LOG_DBG("sem_syncable timed out, resetting");
+	}
+
+	/* sem_broadcast_code_received is also given if the
+	 * broadcast is not encrypted
+	 */
+	LOG_DBG("Waiting for broadcast code");
+
+	err = k_sem_take(&sem_broadcast_code_received, SEM_TIMEOUT);
+	if (err != 0) {
+		LOG_DBG("sem_broadcast_code_received timed out, resetting");
+	}
+
+	values_compare();
+
+	LOG_DBG("Waiting for BIS sync request");
+
+	err = k_sem_take(&sem_bis_sync_requested, SEM_TIMEOUT);
+	if (err != 0) {
+		LOG_DBG("sem_bis_sync_requested timed out, resetting");
+	}
+
+	sync_bitfield = bis_index_bitfield & requested_bis_sync;
+	stream_count = 0;
+	for (int i = 1; i < BT_ISO_MAX_GROUP_ISO_COUNT; i++) {
```

Comment:
This starts at 1?

Reply:
There has been some back and forth with the ISO indexes starting at 0 or 1, can't remember if it changed to being 0 or 1 in the latest change. It is still set to start at 1 in the broadcast_sink sample in Zepyr, where this code is taken from. 
It doesn't matter for this test though, as we are only looking at the adv data.

Reply:
ISO indexes start at 1. 
We did however change the bitfield so that BIT(0) == Index 1, where previously BIT(1) == Index 1